doc/source/quickstart.rst

```diff
@@ -130,6 +130,20 @@ To access the loaded data use the wavelength or name:
 For more information on loading datasets by resolution, calibration, or other
 advanced loading methods see the :doc:`readers` documentation.
 
+
+Calculating measurement values and navigation coordinates
+=========================================================
+
+Once loaded, measurement values can be calculated from a DataArray within a scene, using .values to get a fully calculated numpy array:
```

Comment:
Should we add a warning about potential memory problems ?

Reply:
Also added a note about this, along with example (albeit not quantified beyond "gigabytes").